observation_portal/settings.py

```diff
@@ -187,21 +187,21 @@
 # https://docs.djangoproject.com/en/2.1/howto/static-files/
 
 AWS_STORAGE_BUCKET_NAME = os.getenv('AWS_BUCKET_NAME', 'observation-portal-test-bucket')
-AWS_REGION = os.getenv('AWS_REGION', 'us-west-2')
+AWS_S3_REGION_NAME = os.getenv('AWS_REGION', 'us-west-2')
 AWS_ACCESS_KEY_ID = os.getenv('AWS_ACCESS_KEY_ID', None)
 AWS_SECRET_ACCESS_KEY = os.getenv('AWS_SECRET_ACCESS_KEY', None)
-AWS_S3_CUSTOM_DOMAIN = 's3-us-west-2.amazonaws.com/{}'.format(AWS_STORAGE_BUCKET_NAME)
 AWS_IS_GZIPPED = True
 AWS_LOCATION = os.getenv('MEDIAFILES_DIR', 'media')
 AWS_DEFAULT_ACL = None
+AWS_S3_SIGNATURE_VERSION = 's3v4'
 
 STATICFILES_DIRS = [os.path.join(BASE_DIR, 'static')]
 STATIC_URL = '/static/'
 STATIC_ROOT = '/static/'
 STATICFILES_DIRS = [os.path.join(BASE_DIR, 'static')]
 STATICFILES_STORAGE = os.getenv('STATIC_STORAGE', 'django.contrib.staticfiles.storage.StaticFilesStorage')
 MEDIAFILES_DIR = os.getenv('MEDIAFILES_DIR', 'media')
-MEDIA_URL = "https://{}/{}/".format(AWS_S3_CUSTOM_DOMAIN, MEDIAFILES_DIR) if AWS_ACCESS_KEY_ID else '/media/'
+MEDIA_URL = '' if AWS_ACCESS_KEY_ID else '/media/'
```

Comment:
I'm guessing we just don't even use MEDIA_URL anymore since we don't use templates? Previously we used this to determine if it should use AWS for media files, or local media file hosting for dev - I'm guessing the local hosting doesn't work anymore in the frontend since it's probably just pointing to s3 now?

Reply:
No actually it still works. If you don't set the `MEDIA_STORAGE` environment variable to use the s3 backend, then local file storage will be used by default, and if you have `DEBUG=True`, then the app itself will serve media files at `/media/`, similar to how we handle static files during development.